index.html

```diff
@@ -139,6 +140,17 @@ <h2>Methods</h2>
         </dd>
       </dl>
     </section>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+
+      <p>Append the following paragraph to the description of
+      <a data-cite="WEBRTC#dom-peerconnection-addicecandidate">addIceCandidate</a>:</p>
+
+      <p>If <a data-cite="WEBAPPSEC-CSP#should-block-rtc-connection">
+        should-block-rtc-connection</a> on the current global object returns
```

Comment:
This link doesn't seem to be working. At least not when I hit the Preview link in OP.

Reply:
I assume that's because its target is introduced in https://github.com/w3c/webappsec-csp/pull/457, which hasn't been merged yet.